dts/bindings/spi/st,stm32-spi.yaml

```diff
@@ -0,0 +1,42 @@
+---
+title: STM32 SPI
+id: st,stm32-spi
+version: 0.1
+
+description: >
+    This binding gives a base representation of the STM32 SPI controller
```

Comment:
There is at least one important difference in the SPI IP blocks found on the different families, namely whether or not a FIFO is available for I/O or if just a single register (the DR) can be used. Does that need to be reflected here?

Reply:
(I'm not saying it does, just asking)